modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-major.adoc

```diff
@@ -0,0 +1,359 @@
+= Anypoint MQ Connector 3.x Migration Guide - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint MQ connector version 3.0 includes new features, such as simplified configuration, standardized endpoints, and non-blocking functionality.
+
+=== Global Configuration
+
+In version 2.x of this connector, the `config` element included parameters, as shown in this example:
+
+[source,xml,linenums]
+----
+<anypoint-mq:default-subscriber-config
+    name="Anypoint_MQ_Default_subscriber"
+    acknowledgementMode="IMMEDIATE"
+    acknowledgementTimeout="10000" <!-- (1) -->
+    maxRedelivery="2"              <!-- (2) -->
+    pollingTime="2000"
+    fetchSize="5"                  <!-- (3) -->
+    fetchTimeout="5000"            <!-- (3) -->
+    frequency="2000">              <!-- (3) -->
+  <anypoint-mq:connection          <!-- (4) -->
+    clientId="${client.id}"
+    clientSecret="${client.secret}"/>
+  <anypoint-mq:circuit-breaker     <!-- (5) -->
+    circuitName="InvoiceSettings"
+    onErrorTypes="JMS:TIMEOUT"
+    errorsThreshold="10" tripTimeout="1"
+    tripTimeoutUnit="MINUTES" />
+</anypoint-mq:default-subscriber-config>
+----
+
+In version 3.x of this connector, the `config` element for the MQ Connector now contains only the connection settings:
+
+[source,xml,linenums]
+----
+<anypoint-mq:config name="Anypoint_MQ_Config">
+    <anypoint-mq:connection
+        clientId="${client.id}"
+        clientSecret="${client.secret}" />
+</anypoint-mq:config>
+----
+
+This table describes the `config` parameter differences.
+
+[%header,cols="20a,40a,40a"]
+|===
+|Element | Version 2.x Behavior | Version 3.x Equivalent
+|*(1)* - Acknowledgment
+|Global acknowledgment settings for the Anypoint MQ Subscribers
+|Configure for each Subscriber instance.
+
+|*(2)* - Max Redelivery
+|Redelivery filter for Anypoint MQ messages located in the `redeliveryCount` header
```

Comment:
Period.
I have seen the error of my ways.... If we cannot have all fragments...if there is even one sentence in a column...then we should end all entries with periods.